.changes/1996-_dependents.isEmpty-is-not-true.md

```diff
@@ -0,0 +1 @@
+- Fix UI bug - '_dependents.isEmpty': is not true. We can see this error in red screen at chat room page.
```

Comment:
this isn't a useful message to an end-user. better leaving the file empty in this case IMHO.
```suggestion
```